tests/bluetooth/controller/mock_ctrl/CMakeLists.txt

```
@@ -0,0 +1,35 @@
+#
+# CMakeLists.txt file for creating of mocks library.
+#
+
+add_library(mocks STATIC
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/kernel.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ecb.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/mayfly.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/lll.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/lll_conn.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ll_assert.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/assert.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/util.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ticker.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ull.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ull_conn_iso.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ull_peripheral.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ull_peripheral_iso.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ull_central.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/ull_scan.c
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/src/lll_clock.c
+)
+
+
+target_include_directories(mocks PUBLIC
+  ${ZEPHYR_BASE}/tests/bluetooth/controller/mock_ctrl/include
```

this line can be reduced to
```suggestion
  include
```
as that include folder is below the `CMAKE_CURRENT_SOURCE_DIR`.
This also makes it clearer to the reader which folders are local to the mock lib, and which folders are picked up externally (as they use abs path)

---

done